README.md

```diff
@@ -1,10 +1,21 @@
-# rustls-openssl-compat
+# Rustls OpenSSL Compatibility Layer
 
-This is the planned home of several rustls to OpenSSL compatibility layers.
-Currently here:
+This is an implementation of the OpenSSL libssl ABI which uses [Rustls](https://github.com/rustls/rustls/) for TLS.
 
-- **[rustls-libssl](rustls-libssl/)**: an implementation of the OpenSSL libssl ABI in terms of rustls.
```

Comment:
would be good to retain this link -- somewhere -- as it has a fuller readme that this one. or maybe we should move that file to here? See https://github.com/rustls/rustls-openssl-compat/tree/main/rustls-libssl